pkg/util/k8s/kubeproxy.go

```diff
@@ -68,7 +68,6 @@ func GenerateKubeProxyConfiguration(args map[string]operv1.ProxyArgumentList) (s
 
 	kpc.UDPIdleTimeout.Duration = ka.getDuration("udp-timeout")
 
-	kpc.Conntrack.Max = ka.getOptInt32("conntrack-max")
```

Comment:
I assume the unit tests are going to need to be updated to reflect this field going away. It might be good to add an explicit test that we get an error now if the user sets `conntrack-max` in the config.

Reply:
yah, i'm actually cleaning up text matrixes setting up max.
Should've really put WIP on the BZ, will amend